docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -55,7 +58,19 @@ HOST_MOUNT_DIR=/data/dockerMount
 [[ -h /data/srv/wmagent ]] && rm -f /data/srv/wmagent
 ln -s $HOST_MOUNT_DIR/srv/wmagent /data/srv/wmagent
 
+# create the passwd and group mount point dynamically at runtime
+passwdEntry=`getent passwd $wmaUser  |awk -F : -v wmaUser=$wmaUser '{print $1":"$2":"$3":"$4":"$5":""/home/"wmaUser":"$7}'`
+groupEntry=`getent group $wmaGroup`
 
+[[ -d $HOST_MOUNT_DIR/admin/ ]] || (mkdir -p $HOST_MOUNT_DIR/admin) || exit $?
+
+getent passwd > $HOST_MOUNT_DIR/admin/passwd
```

Comment:
Ok @vkuznet , I made a minor mistake here. I just found  it myself. Here the 3 containers would overlap and would overwrite their files one another..... so we should have this line for the 3 container flavours in  3 different configurations ( with the respective sources for the mount points bellow):

```
[[ -d $HOST_MOUNT_DIR/admin/wmagent/etc ]] || (mkdir -p $HOST_MOUNT_DIR/admin/wmagent/etc) || exit $?
getent passwd > $HOST_MOUNT_DIR/admin/wmagent/etc/passwd
```

Reply:
Nope, exactly the opposite: 

We should recreate the them 3 times in the 3 different hostmount  areas:

* docker/pypi/wmagent-couchdb/couchdb-docker-run.sh
```
[[ -d $HOST_MOUNT_DIR/admin/couchdb/etc ]] || (mkdir -p $HOST_MOUNT_DIR/admin/couchdb/etc) || exit $?
getent passwd > $HOST_MOUNT_DIR/admin/couchdb/etc/passwd
...
--mount type=bind,source=$HOST_MOUNT_DIR/admin/couchdb/etc/passwd,target=/etc/passwd,readonly \
```
* docker/pypi/wmagent-mariadb/mariadb-docker-run.sh
```
[[ -d $HOST_MOUNT_DIR/admin/mariadb/etc ]] || (mkdir -p $HOST_MOUNT_DIR/admin/mariadb/etc) || exit $?
getent passwd > $HOST_MOUNT_DIR/admin/mariadb/etc/passwd
...
--mount type=bind,source=$HOST_MOUNT_DIR/admin/mariadb/etc/passwd,target=/etc/passwd,readonly \
```
* docker/pypi/wmagent/wmagent-docker-run.sh

```
[[ -d $HOST_MOUNT_DIR/admin/wmagent/etc ]] || (mkdir -p $HOST_MOUNT_DIR/admin/wmagent/etc) || exit $?
getent passwd > $HOST_MOUNT_DIR/admin/wmagent/etc/passwd
...
--mount type=bind,source=$HOST_MOUNT_DIR/admin/wmagent/etc/passwd,target=/etc/passwd,readonly \
```


But I am not sure if we are not to take the other path @amaltaro  was  suggesting here: https://github.com/dmwm/CMSKubernetes/pull/1468 which should in general make this step unnneded